docs/uninstall.md

```
@@ -0,0 +1,188 @@
+
+```yaml
+packages:
+  my-package:
+    version: "1.0.0"
+    image: ghcr.io/example/pkg
+    uninstall:
+      enabled: true
+      apply: true      # triggers uninstall
+```
+
+2. The operator creates uninstall pods on each target node. These pods run with the **full package configuration** (ConfigMap, env, resources) — not a synthetic stub.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)